builtins/amp-ad.js

```diff
@@ -217,6 +223,12 @@ export function installAd(win) {
       assert(!this.isInFixedContainer_,
           '<amp-ad> is not allowed to be placed in elements with ' +
           'position:fixed: %s', this.element);
+      if (this.isResizable_) {
+        this.element.setAttribute('scrolling', 'no');
+        assert(this.getOverflowElement(),
```

Comment:
I like that suggestion! Lets land this and follow up with that.

CC @kashyapnitin